### DISCUSSION TITLE
Is Insiders free for those who contribute to this project?

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#5433] [Q&A] [answered]
Hey @squidfunk,
Is Insiders free for those who contribute to this project? It could be simple change such as typo error or may be adding new language support etc.

If no, any plans to implement this year such that folks or organisations can support not only by sponsoring monetarily but also possible by sharing their skills and knowledge?

**Comment by squidfunk** [comment]
Great question! Currently no plans, no. Sponsorships help us maintainers set aside enough time to push the project forward and help our users to succeed in what they're trying to achieve by answering questions. From your question I understand you're asking about "drive-by" contributions, which means more for us to review, thus more time invested, so nothing we can give free access for. 

With increasing funds, we're starting to look for individuals to support the project not as part-time or drive-by contributors, but to be part of the core team, of course with financial compensation. For that to work, we need individuals with experience in the following technologies and paradigms:

- Deep knowledge of CSS, HTML, TypeScript
- Experience with progressive enhancement and responsive design
- Experience with reactive programming with RxJS
- Solid knowledge of Python, MkDocs + ecosystem
- Solid technical writing skills

If you or somebody else is up for this and would love to work with us on this project, please contact us at sponsors@squidfunk.com.

**Reply by squidfunk** [reply]
One thing I want to add: [we are already upstreaming some of our funds to MkDocs plugin and extension developers](https://github.com/squidfunk?tab=sponsoring) that we rely on and/or that are natively supported by Material for MkDocs. If somebody maintains a popular extension, we're happy to include the maintainer in our upstream funding as well. A GitHub Sponsors account is a requirement. We have also offered this to several individuals, some of which are _regularly_ answering questions here on our discussion board with great detailed. The upstream funding will definitely cover the cost of Insiders, so it's more or less what you asked for ☺️

**Reply by ghost** [reply]
What about the projects that are open source or NPO, @squidfunk ; Is there anyway they can apply to get insiders features for free i.e without sponsorship or be maintainers?

**Reply by squidfunk** [reply]
For Open Source projects and non-profits that do not want to or cannot pay for additional features in Insiders, the community edition should already be powerful enough. All features that are in the community edition will remain free, forever. Material for MkDocs is one of the most powerful solutions to build technical documentation without any knowledge of web technologies. We support and maintain the community edition for free, and partially compensate for the time we invest via Insiders subscriptions.

**Comment by katharinalisalin** [comment, marked as answer]
__Exciting Update for Enthusiasts & Contributors!__
If you're discovering this discussion later on and are also curious about ways to engage with and contribute to our project, we have some thrilling news for you. After much consideration on how to involve our users more deeply in the project and to create a way to provide Insiders access to those who might not be able to obtain it otherwise, we're proud to introduce our [Community Experts Program](https://squidfunk.github.io/mkdocs-material/insiders/community-experts-program/).

This program is for our users who are passionate about Material for MkDocs and keen on sharing their knowledge and assisting others within our community on [our discussion board](https://github.com/squidfunk/mkdocs-material/discussions). Your role as a Community Expert involves __helping fellow members with their questions__, __offering insights__ on using Material for MkDocs effectively and __sharing tips on workarounds__, __alternative setups__, and __minor customizations__. Becoming a Community Expert is the best way to create a significant impact and deliver the most meaningful contribution to this project.

As a token of appreciation for your invaluable contributions, we offer __complimentary access to the Insiders edition__ for our active Community Experts. If this opportunity excites you, we encourage you to visit our [Community Experts page](https://squidfunk.github.io/mkdocs-material/insiders/community-experts-program/) to learn more about how you can get involved.
